packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/types.ts

```diff
@@ -0,0 +1,34 @@
+import { AssetPropertySummary, AssetModelPropertySummary } from '@aws-sdk/client-iotsitewise';
+
+type listPropertyOutputExtender = {
+  externalId?: string;
+  path?: [
+    {
+      id: string;
+      name: string;
+    }
+  ];
+  unit?: string;
+};
+
+type newAssetPropertySummary = AssetPropertySummary & listPropertyOutputExtender;
+
+export type newAssetModelPropertySummary = AssetModelPropertySummary & listPropertyOutputExtender;
+export type newListAssetPropertiesCommandOutput = {
+  assetPropertySummaries: newAssetPropertySummary[] | undefined;
+};
+
+export type newListAssetModelPropertiesCommandOutput = {
+  assetModelPropertySummaries: newAssetModelPropertySummary[] | undefined;
+};
+
+export type SelectedAssetProp = {
+  assetId: string;
+  assetModelId: string;
+};
+
+export type SelectedAssetsPropsArray = SelectedAssetProp[];
```

Comment:
Can this just be `SelectedAsset`? Appending `Prop` to the end doesn't seem to add meaning and may actually add confusion.